exercise-1/script.js

```diff
@@ -0,0 +1,22 @@
+const button = document.querySelector('button');
+const nav = document.querySelector('nav');
+const main = document.querySelector('.hide-main');
+
+button.addEventListener('click', function () {
+  nav.classList.toggle('display');
+  main.classList.toggle('hide-main');
```

Comment:
I'd like to challenge you here. Would you be able to achieve the same result by toggling the class name only on the `nav`, and not on the `main`? You can do it with some CSS trickery.

Hint: [Adjacent sibling combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator)